c++/src/capnp/compat/json.h

```diff
@@ -313,6 +313,28 @@ class JsonCodec::Handler<T, Style::POINTER>: private JsonCodec::HandlerBase {
   friend class JsonCodec;
 };
 
+template <>
+class JsonCodec::Handler<DynamicValue>: private JsonCodec::HandlerBase {
```

Comment:
That does sound better -- I'll give it a shot tonight or tomorrow and update the branch.